docs/tutorials/zarr-v3.md

```diff
@@ -0,0 +1,101 @@
+# zarr-v3 Guide/Roadmap
+
+`anndata` now uses the much improved {mod}`zarr` v3 package and also allows writing of datasets in the v3 format via {attr}`anndata.settings.zarr_write_format`, with the exception of structured arrays.
+Users should notice a significant performance improvement, especially for cloud data, but also likely for local data as well.
+Here is a quick guide on some of our learnings so far:
+
+## Remote data
+
+We now provide the {func}`anndata.experimental.read_lazy` feature for reading as much of the {class}`~anndata.AnnData` object as lazily as possible, using `dask` and {mod}`xarray`.
+Please note that this feature is experimental and subject to change.
+To enable this functionality in a performant and feature-complete way for remote data sources, we use {doc}`conslidated metadata <zarr:user-guide/consolidated_metadata>` on the `zarr` store (written by default).
+Please note that this introduces consistency issues – if you update the structure of the underlying `zarr` store i.e., remove a column from `obs`, the consolidated metadata will no longer be valid.
+Further, note that without consolidated metadata, we cannot guarantee your stored `AnnData` object will be fully readable.
+And even if it is fully readable, it will almost certainly be much slower to read.
+
+There are two ways of opening remote `zarr` stores from the `zarr-python` package, {class}`zarr.storage.FsspecStore` and {class}`zarr.storage.ObjectStore`, and both can be used with `read_lazy`.
+[`obstore` claims] to be more performant out-of-the-box, but notes that this claim has not been benchmarked with the `uvloop` event loop, which itself claims to be 2× more performant than the default event loop for `python`.
+
+## Local data
+
+Local data generally poses a different set of challenges.
+First, write speeds can be somewhat slow and second, the creation of many small files on a file system can slow down a filesystem.
+For the "many small files" problem, `zarr` has introduced `{ref} sharding <zarr:user-guide/performance#sharding>` in the v3 file format.
```

Comment:
Why not link to the `user-guide-sharding` label? It’s a different “Sharding” headline, but I think it contains the same information.

Reply:
> If you do, please don’t customize the link text, it already reads “Sharding”.

Reply:
> If you do, please don’t customize the link text, it already reads “Sharding”.

not a huge fan of the capitalization...